Detectors/ITSMFT/common/simulation/include/ITSMFTSimulation/Digitizer.h

```diff
@@ -59,7 +59,7 @@ class Digitizer : public TObject
   o2::itsmft::DigiParams& getParams() { return (o2::itsmft::DigiParams&)mParams; }
   const o2::itsmft::DigiParams& getParams() const { return mParams; }
   void setNoiseMap(const o2::itsmft::NoiseMap* mp) { mNoiseMap = mp; }
-  void setDeadChannelsMap(const o2::itsmft::NoiseMap* mp) { mDeadChanMap = mp; }
+  void setDeadChannelsMap(o2::itsmft::NoiseMap* mp) { mDeadChanMap = mp; }
```

Comment:
What is the reason to remove the constness? Since the digitizer does not modify the map, it is better to keep it const.